.editorconfig

```diff
@@ -0,0 +1,21 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{yml,yaml,sh}]
```

Comment:
I suggest separating `yaml` rule and `sh` rule. Just like `Jenkinsfile`.

Reply:
Done